includes/api/save-abandonment-data.php

```diff
@@ -101,6 +101,40 @@ function saveCartAbandonmentData(WP_REST_Request $request)
         }
     }
 
+    // check yith woocommerce recover abandoned cart plugin activated or not
+    if (is_plugin_active('yith-woocommerce-recover-abandoned-cart-premium/init.php') && (empty($razorpayData['customer_details']['email']) == false) && (empty($razorpayData['customer_details']['shipping_address']))== true) {
```

Comment:
Can you move this to support folder? As we're moving all the supported plugins to support folder.